Makefile

```diff
@@ -0,0 +1,24 @@
+# Image can be overidden with an env var.
+DOCKER_IMAGE?=microscaling/microscaling
+
+# Version tag from the code.
+VERSION=`cat VERSION`
```

Comment:
Maybe there could be a makefile target for building the latest tagged version (maybe cat the VERSION file, use this to retrieve the git code tagged as that version number). And another target that tags it differently if it's a development build (maybe the tag includes the latest SHA?)